x-pack/plugins/security_solution/public/detections/components/rules/related_integrations/use_installed_integrations.tsx

```diff
@@ -34,6 +36,7 @@ export const useInstalledIntegrations = ({ packages }: UseInstalledIntegrationsA
     },
     {
       keepPreviousData: true,
+      staleTime: ONE_MINUTE * 15,
```

Comment:
Hmm, from the APM measurements that I saw, it looked like the installed integrations requests were not hitting cache for some reason 🤔. The data showed that the requests were taking ~500ms, which is pretty substantial. Also, they were triggered after random user actions. I suppose every time the Rules table is mounted, like after navigating to a rule page and then going back. So that's why I thought caching those requests could help minimize pressure on Kibana, considering that integrations are not something that changes frequently.

That said, I'm okay with setting the stale time to 5 minutes. It should be enough to lower the number of outgoing requests and not harm the user experience.